.github/workflows/build.yml

```diff
@@ -107,8 +107,7 @@ jobs:
         # --scan:         Attempt to publish build scans in PRs. This will only work on PRs from apache/kafka, not public forks.
         # --continue:     Keep running even if a test fails
         # -PcommitId      Prevent the Git SHA being written into the jar files (which breaks caching)
-        timeout-minutes: 180  # 3 hours
-        continue-on-error: true
+        timeout-minutes: 10  # 3 hours
```

Comment:
I guess this should be reverted?

Reply:
Yes, I was testing something unrelated to this PR. I've noticed timeouts are showing a green check mark in the job overview which is misleading.